PolyEngine/UnitTests/Src/QueueTests.cpp

```diff
@@ -9,47 +9,48 @@ TEST_CASE("Queue tests", "[Queue]")
 	Queue<int> q;
 	Queue<int> r(q);
 
-	REQUIRE(q.Empty() == true);
-	REQUIRE(r.Empty() == true);
+	REQUIRE(q.IsEmpty() == true);
```

Comment:
It would be very nice to have more elaborate unit tests that check for example various constructors and assign operators, resizes and reserves.